pkg/query-service/app/server.go

```diff
@@ -257,7 +258,15 @@ func (s *Server) Start() error {
 			zap.S().Error("Could not start HTTP server", zap.Error(err))
 		}
 		s.unavailableChannel <- healthcheck.Unavailable
+	}()
+
+	go func() {
+		zap.S().Info("Starting pprof server", zap.String("addr", constants.DebugHttpPort))
 
+		err = http.ListenAndServe(constants.DebugHttpPort, nil)
+		if err != nil {
+			zap.S().Error("Could not start HTTP server", zap.Error(err))
```

Comment:
Maybe update this error to `Could not start pprof HTTP server` ?

Reply:
Done